### DISCUSSION TITLE
大家切号的时候古树是怎么操作的？

### DISCUSSION THREAD

**Discussion by Zhangfanyu1222** [constanline/XQuickEnergy#371] [General/我有一言当讲否]
宝汁们在切号之前，都会记得把古树开关先关闭再切号吗？:joy:忘记关闭了就会嚯嚯了...还是说各位是手动点古树:interrobang:

**Comment by alinnse** [comment]
不切号，用多开

**Comment by Liujishou** [comment]
https://github.com/buddingworld/XQuickEnergy
这位大佬做了大小号功能，不过没开源！可以下载打包好的文件使用。

**Reply by Zhangfanyu1222** [reply]
没开源的不敢用啊😂，上次一个人用了打包好的，好友被删光，能量被花光
![图片](https://github.com/constanline/XQuickEnergy/assets/65014915/7dd148ab-fe3c-4092-8785-2cecec604da6)
